deploy_nixos/nixos-instantiate.sh

```diff
@@ -10,7 +10,17 @@ shift
 shift
 
 # Building the command
-command=(nix-instantiate "<nixpkgs/nixos>")
+command=(nix-instantiate --show-trace --expr '
+  { system, configuration, ... }:
```

Comment:
is `system` auto-filled? I'm not seeing it passed to the command args.

Reply:
That still works the same. It is passed here: https://github.com/tweag/terraform-nixos/pull/8/files#diff-58e561a5cf45d2adc3bb3a4abe7efd64R78

Reply:
thanks!